demo/addons/godotopenxrmeta/godot_openxr_meta_export_plugin.gd

```diff
@@ -0,0 +1,207 @@
+@tool
+extends EditorExportPlugin
+
+const OPENXR_MODE_VALUE = 1
+
+const PASSTHROUGH_NONE_VALUE = 0
+const PASSTHROUGH_OPTIONAL_VALUE = 1
+const PASSTHROUGH_REQUIRED_VALUE = 2
+
+const HAND_TRACKING_NONE_VALUE = 0
+const HAND_TRACKING_OPTIONAL_VALUE = 1
+const HAND_TRACKING_REQUIRED_VALUE = 2
+
+const HAND_TRACKING_FREQUENCY_LOW_VALUE = 0
+const HAND_TRACKING_FREQUENCY_HIGH_VALUE = 1
+
+const ENABLE_META_LOADER_OPTION = {
+		"option": {
+			"name": "xr_features/enable_meta_plugin",
+			"class_name": "",
+			"type": TYPE_BOOL,
+			"hint": PROPERTY_HINT_NONE,
+			"hint_string": "",
+			"usage": PROPERTY_USAGE_DEFAULT,
+		},
+		"default_value": false,
+		"update_visibility": false,
+	}
+	
+const HAND_TRACKING_OPTION = {
+	"option": {
+		"name": "meta_xr_features/hand_tracking",
+		"class_name": "",
+		"type": TYPE_INT,
+		"hint": PROPERTY_HINT_ENUM,
+		"hint_string": "None,Optional,Required",
+		"usage": PROPERTY_USAGE_DEFAULT,
+	},
+	"default_value": HAND_TRACKING_NONE_VALUE,
+	"update_visibility": false,	
+}
+
+const HAND_TRACKING_FREQUENCY_OPTION = {
+	"option": {
+		"name": "meta_xr_features/hand_tracking_frequency",
+		"class_name": "",
+		"type": TYPE_INT,
+		"hint": PROPERTY_HINT_ENUM,
+		"hint_string": "Low,High",
+		"usage": PROPERTY_USAGE_DEFAULT,
+	},
+	"default_value": HAND_TRACKING_FREQUENCY_LOW_VALUE,
+	"update_visibility": false,
+}
+
+const PASSTHROUGH_OPTION = {
+		"option": {
+			"name": "meta_xr_features/passthrough",
+			"class_name": "",
+			"type": TYPE_INT,
+			"hint": PROPERTY_HINT_ENUM,
+			"hint_string": "None,Optional,Required",
+			"usage": PROPERTY_USAGE_DEFAULT,
+		}, 
+		"default_value": PASSTHROUGH_NONE_VALUE,
+		"update_visibility": false,
+	}
+
+
+func _get_name():
+	return "GodotOpenXRMeta"
+
+
+# For production, the Maven central dependency is used.
+# This should be commented out if '_get_android_libraries' is enabled
+func _get_android_dependencies(platform, debug):
+	if platform is EditorExportPlatformAndroid and _is_meta_loader_option_enabled():
```

Comment:
Addressed both concerns:
- Added a method to check if the aar file is available; if it is the plugin uses it, otherwise it falls back to the maven central dependency
- Added a `get_plugin_version()` method to the `EditorExportPlugin` so we can retrieve the value